MonoGame.Framework/SDL/SDLGameWindow.cs

```diff
@@ -157,33 +179,32 @@ internal void CreateWindow()
             Sdl.Window.SetResizable(_handle, _resizable);
 
             SetCursorVisible(_mouseVisible);
+
+            _supressMoved = true;
         }
 
         ~SdlGameWindow()
         {
             Dispose(false);
         }
 
-        private static Sdl.Rectangle GetMouseDisplay()
+        private static int GetMouseDisplay()
```

Comment:
The only reason to use this method is to get the mouse rectangle, so by returning the int you are making 1 extra wasteful call to the native API later on.